clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -175,7 +175,8 @@ Changes in existing checks
 - Improved :doc:`bugprone-sizeof-expression
   <clang-tidy/checks/bugprone/sizeof-expression>` check to find suspicious
   usages of ``sizeof()``, ``alignof()``, and ``offsetof()`` when adding or
-  subtracting from a pointer directly or when used to scale a numeric value.
+  subtracting from a pointer directly or when used to scale a numeric value and
+  fix false positive when sizeof expression with template types.
```

Comment:
If there were no releases since this change, then there is no need for the release note entry. Basically you're hotfixing a change that is coming out in the same hypothetical next release.

If the change in this patch is not related to what is mentioned in this changelog entry, then it is worth a full new changelog entry, like this:

```suggestion
  subtracting from a pointer directly or when used to scale a numeric value.
  
- Fixed :doc:`bugprone-sizeof-expression
  <clang-tidy/checks/bugprone/sizeof-expression>` check not to report on
  ``sizeof()/sizeof()`` expressions for template types.
```

Reply:
I think not. clang-tidy release-note always combines the all changes of rule in one item instead of spilt them.

Reply:
example: 
`readability-container-contains`
`readability-enum-initial-value`

Reply:
> If there were no releases since this change, then there is no need for the release note entry. Basically you're hotfixing a change that is coming out in the same hypothetical next release.

I'm not sure what you meant here. There should be a release note, unless the issue was introduced in the current, release cycle. The diagnostic reproduces in clang-tidy 18 for me, or did you mean something different (e.g., general *in-case-of* comment)?

> clang-tidy release-note always combines the all changes of rule in one item instead of spilt them.

+1, that's the current approach

Reply:
> > If there were no releases since this change, then there is no need for the release note entry. Basically you're hotfixing a change that is coming out in the same hypothetical next release.
> 
> I'm not sure what you meant here. There should be a release note, unless the issue was introduced in the current, release cycle. The diagnostic reproduces in clang-tidy 18 for me, or did you mean something different (e.g., general _in-case-of_ comment)?

Because of the merged release note I thought this is related to #106061, hence the confusion.